src/test/java/byteceps/processing/WeeklyProgramManagerTest.java

```diff
@@ -378,4 +405,34 @@ void log_historyInvalidDate_throwsInvalidInput() {
         parser.parseInput(logHistoryInput);
         assertThrows(Exceptions.InvalidInput.class, () -> weeklyProgramManager.execute(parser));
     }
+
+    @Test
+    void execute_list_success() {
+        setUpStreams();
+        String assignWorkoutInput = "program /assign full day /to monday";
+        parser.parseInput(assignWorkoutInput);
+        assertDoesNotThrow(() -> ui.printMessage(weeklyProgramManager.execute(parser)));
+
+        String listInput = "program /list";
+        parser.parseInput(listInput);
+        assertDoesNotThrow(() -> ui.printMessage(weeklyProgramManager.execute(parser)));
+
+        String expectedOutput = "[BYTE-CEPS]>Workoutfulldayassignedtomonday\n" +
+                "-------------------------------------------------";
+
+        expectedOutput += "[BYTE-CEPS]> Your workouts for the week:\n" +
+                "\tMONDAY: full day\n" +
+                "\tTUESDAY: Rest day\n" +
+                "\tWEDNESDAY: Rest day\n" +
+                "\tTHURSDAY: Rest day\n" +
+                "\tFRIDAY: Rest day\n" +
+                "\tSATURDAY: Rest day\n" +
+                "\tSUNDAY: Rest day\n" +
+                "-------------------------------------------------";
```

Comment:
Would it be better to use values from DayStrings here and implement this with String.format? (this is a genuine question because your current implementation works better for readability and other tests do also use this implementation, but Joshua also refactored many things into strings package, so I'm not sure)

Reply:
Hmm I think in this case it does not really make sense to go through another layer of grabbing values from DayStrings as it is constant.